src/components/crownConfig.vue

```diff
@@ -23,24 +23,26 @@
       :title="$t('Message Flow')"
     />
 
-    <div class="dropdown-container">
-      <button class="button" @click="dropdownOpen = !dropdownOpen">
+    <div class="cog-container" role="menubar">
+      <button class="cog-container--button" @click="dropdownOpen = !dropdownOpen">
         <i class="fas fa-cog"/>
       </button>
 
-      <ul class="dropdown" v-if="dropdownOpen">
-        <li>
+      <ul class="element-list" v-if="dropdownOpen" role="list">
+        <li class="element-list--item" role="listitem">
```

Comment:
The `li` element already has a default role of `listitem`, but considering my comment above, it may be best to explifictly define the role anyway.

I dug into it a little more more and came across this article: https://inclusive-components.design/menus-menu-buttons/ (the other articles on that website are also great). That page gives the following example:

```html
<button aria-haspopup="true" aria-expanded="false">
  Difficulty
  <span aria-hidden="true">&#x25be;</span>
</button>
<div role="menu">
  <button role="menuitem" tabindex="-1">Easy</button>
  <button role="menuitem" tabindex="-1">Medium</button>
  <button role="menuitem" tabindex="-1">Incredibly Hard</button>
</div>
```

https://www.w3.org/TR/wai-aria-1.1/#menu
https://www.w3.org/TR/wai-aria-1.1/#menuitem